js/everything.js

```diff
@@ -427,6 +428,30 @@ function anchorURLs(text) {
     return text.replace(re, "<a href=\"$1\" target=\"_blank\">$1</a>");
 }
 
+function anchorTimestamps(text, videoID) {
+    /* RegEx to match
+      hh:mm:ss
+      h:mm:ss
+      mm:ss
+      m:ss
+    and wraps the timestamps in <a> tags
+    */
+    var re = /((?:[0-5]\d|\d|)(?:\d|\:[0-5]\d)(?:$|\:[0-5]\d))/g;
+    return text.replace(re, (match, $1) => "<a href=\"" + makeListenURL(videoID, convertTimestamp($1)) + "\">" + $1 + "</a>");
+}
+
+function convertTimestamp(stamp) {
+    // changes timestamps from 01:00:00 to 01h00m00s for example for the makeListenURL function
```

Comment:
The logic here is wrong. The `t` querystring parameter is the time in seconds, ie: `t=120` for 2 minutes. The return value of this function should be an integer